tests/scheduler/test_utils.py

```diff
@@ -59,7 +60,6 @@ def test_example(self):
             "pair_15, ri, b",
             "pair_15, yy, a",
             "pair_15, yy, b",
-            "pair_15, zy, a",
```

Comment:
Why not check for this one?

Reply:
I wasn't seeing it come up. It may have been part of different changes .. I'll check what's being scheduled and put it back if it's present.